app/services/payments/customer.rb

```diff
@@ -65,11 +65,14 @@ def charge(customer:, amount:, description:, card_id: nil)
       def request
         yield
       rescue Stripe::InvalidRequestError => e
+        DatadogStatsClient.increment("stripe.errors")
```

Comment:
Lets break these up like the ones above with tags to get more granular and then we can parse them in Datadog if we want. 
```ruby
DatadogStatsClient.increment("stripe.errors", tags: ["error: InvalidRequestError", "event:charge"])
```

Reply:
What would be the event for `CardError` and `StripeError`, `"charge"` as well? Are those errors only raised for a specific reason?

Reply:
looks like these can come from different methods, so maybe dont include the event unless you want to try and infer the method name that raise the error with `__method__` or something